Evolutionary_loop/Antenna_Performance_Metric/XFintoARA.py

```diff
@@ -0,0 +1,151 @@
+#*******************************************************************************
+#  File: XFintoARA.py
+#     This script translates XF output into readable AraSim input
+#
+#  Programmer: Jorge Torres
+#              Suren Gourapura
+#
+#  Revision history:
+#     11/??/17 Original version
+#     02/21/19 Suren, modified from the old XFintoARA.py
+#     07/19/23 Jason Yao, merged XFintoARA2.py into this script because
+#              the latter contains no significant difference (see line 87)
+#
+#  Notes:
+#     * vertical ruler at column 80
+#
+#  TODO:
+#
+#*******************************************************************************
+'''
+Have all XF output files in the same folder (see uanLoc) with the standard name:
+    "antennaNumber_frequencyNumber.uan"
+Both of these numbers start counting on 1, not zero.
+
+Will output one file (see datLoc) that combines all frequencies,
+so one dat file per antenna. 
+The dat standard name is: 
+    "evol_antenna_model_(i).dat" 
+where (i) is the antenna number starting from 1, not zero.
+
+Code Headers:		
+  .uan header: Theta, Phi, Gain Theta (dB), Gain Phi (dB), Phase Theta, Phase Phi
+  .dat header: Theta, Phi, Gain (dB, theta), Gain (theta), Phase (theta)
+
+Usage:
+  python XFintoARA.py (NPOP) (source) (Run name) (generation) (starting individual)
+For example (from the working directory):
+  python Antenna_Performance_Metric/XFintoARA.py 10 . Ryan_test_run3 0 1
+'''
+
+### IMPORTS ###
+import numpy as np  # For array storage, reading and writing files, etc.
+import argparse     # For reading arguments from the terminal or from the bash script
+import os           # For changing the permissions
```

Comment:
```suggestion
import os           # For changing the permissions
import argparse     # For reading command-line arguments

import numpy as np  # For array storage, reading and writing files, etc.
```
Third-party module imports should be separate from stdlib imports by an empty line.

Reply:
Got it!